HesoyamHospital/GraphicEditor/LogIn.xaml.cs

```diff
@@ -37,33 +37,35 @@ public void Button_Click_Map(object sender, RoutedEventArgs e)
             if (!found)
             {
                 found = FindPatient();
-
             }
 
-            if (!found)
-            {
-                MessageBox.Show("Ne postoji korisnik sa unetim podacima.");
-            }
-            else
-            {
-                MainWindow mainWindow = new MainWindow();
-                mainWindow.Show();
-                this.Close();
+                if (!found)
+                {
+                    MessageBox.Show("Ne postoji korisnik sa unetim podacima.");
+                }
+                else 
+                {
+                    MainWindow mainWindow = new MainWindow();
+                    mainWindow.Show();
+                    this.Close();
 
-            }
+                }
 
-        }
+            }
 
 
         public Boolean FindPatient()
         {
             Boolean found = false;
-            foreach (Patient _patient in patients_list)
+            foreach (Patient _patient in patients)
             {
-                if (Username.Text.Equals(_patient.UserName) && Password.Password.Equals(_patient.Password))
```

Comment:
Why is this separated into two conditional statements?
As per me, the code is way more readable when those two conditional statements are merged into one.